data.js

```diff
@@ -0,0 +1,33 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
+  },
+
+  "team-member-section": {
+    "team-member-card": {
+      "image": "/home/susanar/Documentos/Programacion_2021_1/Layouts/my-team-page/photo4.png",
+      "name-title": "Skylah Lu",
+      "charge-text": "UX DESIGNER"
+    }
+  },
+
+  "footer": {
+    "socialMedia":
```

Comment:
If you're using kebab case instead camel case this would be social-media

Reply:
fac9e7d